### ISSUE TITLE
How should we improve the webpage?  Put your ideas here in this thread.

### ISSUE THREAD

**Issue by mrshirts** [livecomsjournal/journal_information#1]
If you have ideas for how to improve the webpage, please make them as comments in this issue thread.

Note that if you have specific changes to text on the website, it should be filed as an issue to the respective Markdown page in this repository.

**Comment by dmzuckerman** [comment]
The opening sentence of the [Author Instructions](https://livecomsjournal.github.io/authors/) page omits the Lessons Learned category:
"LiveCoMS handles several different types of manuscripts including Best Practices Guides, Perpetual Reviews, Comparisons of Molecular Simulation Packages, and Tutorials."

Also a suggestion to improve the site in general.  To the extent possible, I think we should include direct links to the article categories on the topmost/landing pages.  This is for two reasons: (i) I think the article types are unique and hence advertise our journal and encourage further exploration by folks just browsing ... and no one expects to see anything interesting in the 'author instructions' page so it won't be clicked by people browsing; (ii) Most potential authors will be interested in a specific type of article and they would probably prefer to click on the general descriptions of the article types rather than the ever-unenticing author instructions.

**Comment by davidlmobley** [comment]
I'm inclined to agree with all of this. @dmzuckerman - do a PR?

**Comment by dmzuckerman** [comment]
I will start working on a PR in a moment.   @davidlmobley @mrshirts - as I was poking around, I realize we have some unhelpful (pseudo) redundancy: there are two landing pages and two About pages (one each on livecoms.org and github.io).  In neither case is the content the same.  I suggest removing the github.io pages altogether and merging any non-redundant content onto the livecoms.org pages which should be the main page folks will be sent to (by us or by google search).  We don't want to frustrate people with a weird website that will also require more work to update in a perfectly parallel fashion.

**Comment by davidlmobley** [comment]
I agree with the spirit of that, @dmzuckerman , though what "landing page" do you mean on github.io? just the main livecomsjournal.github.io? What would people land on when visiting, then? 

The only other thing I want to make sure is that it is clear to anyone who visits livecomsjournal.github.io directly (for some reason) what exactly the site is there for. :)

**Comment by dmzuckerman** [comment]
Yes, I mean https://livecomsjournal.github.io.  I think that page will create confusion because actually it has the direct links to policies but also says to go to livecoms.org for further info.  Optimally, we have one landing page.

Operationally, it seems to me livecomsjournal.github.io is only a hub to link out to the policies and author instructions ... which already should be linked from livecoms.org.  Put another way, it just creates an extra click for users ... in addition to featuring a different 'About' page from livecoms.org.

Even though the github.io page is nicer and we have more control over it, it seems that livecoms.org is the main site ... unless we want to point livecoms.org to the github.io but I don't think that's going to play right with Scholastica.

**Comment by davidlmobley** [comment]
Yes, I think you're right.

**Comment by dmzuckerman** [comment]
In an effort to recruit student reviewers, I was just going to pass on the [link ](https://livecomsjournal.github.io/about/earlycareer/) to the info ... but it took me quite a few clicks to find it - and of course, I knew it was there somewhere.  So I think it's too buried for real visitors to the site.  This really should get cleaned up.  Should we hire professional help soon??

**Comment by davidlmobley** [comment]
That's probably a good idea. I've not had time to do much lately (paper deadlines, teaching, and trying to get two large collaborative initiatives funded) but am supportive of this.

**Comment by mrshirts** [comment]
I think that this makes sense.  I don't know what our budget is for this
until we get more submissions / donations from other groups, but if we can
get someone to clean things up piecemeal that we can pay for by other
means, that would be useful.

Other update:

CU Boulder libraries will now do the CrossRef registration, so we will just
have to pay for our DOIs. (which is like a dollar a DO1)  My library
contact said they would try to get it set up so we would be able to use it
in the next couple of months.  And they will set it up so we have a journal
level DOI.

**Comment by dmzuckerman** [comment]
Would someone please please please write a brief description of the structure/platform/tools used in generating webpages from github and anything else a potential website consultant would need expertise in?  I want to start shopping this around right away. @mrshirts @ptmerz @davidlmobley ??

We are marketing ourselves as an internet-age journal, but if our website is confusing, that may turn people off at the outset.  I'd like to get website issues addressed ASAP.  I can chip in some money for this.

**Comment by davidlmobley** [comment]
It's a GitHub Pages site, @dmzuckerman ; I think that's the main thing to know (as that's basically all there is to it); probably some Python would be a plus. But you can basically just point someone to the GitHub repo and they will be able to tell exactly what technology it uses from that (one benefit of this is that everything is public/there is nothing hidden on some server somewhere, so one can peer "under the hood" just by looking at the GitHub repo).

**Comment by mrshirts** [comment]
Slightly more description.  It's on Github Pages, and uses a Jekyll theme, adapted from the minimal-mistakes Jekyll theme (https://mmistakes.github.io/minimal-mistakes/), with some custom Python that updates the navigation.
